cfme/cloud/instance.py

```diff
@@ -347,12 +340,6 @@ def create(self, email=None, first_name=None, last_name=None, availability_zone=
             flash.assert_success_message(
                 "VM Provision Request was Submitted, you will be notified when your VMs are ready")
 
-        row_description = 'Provision from [{}] to [{}]'.format(self.template_name, self.name)
```

Comment:
yes. it happens that vm is already provisioned but it's request is still running. this part of code doesn't refresh provider relationships etc. and the idea was to move asserts to tests
it may be leaved but it adds time to run